conan/api/subapi/new.py

```diff
@@ -38,7 +40,8 @@ def get_builtin_template(self, template_name):
                          "bazel_exe": bazel_exe_files,
                          "autotools_lib": autotools_lib_files,
                          "autotools_exe": autotools_exe_files,
-                         "alias": alias_file}
+                         "alias": alias_file,
+                         "local_recipes_index": local_recipes_index_files}
```

Comment:
mmm, running `conan new --help` from this branch doesn't list this new template:

```
% conan new --help
usage: conan new [-h] [-v [V]] [-cc CORE_CONF] [-d DEFINE] [-f] template

Create a new example recipe and source files from a template.

positional arguments:
  template              Template name, either a predefined built-in or a user-provided one. Available built-in templates: basic, cmake_lib, cmake_exe, meson_lib, meson_exe,
                        msbuild_lib, msbuild_exe, bazel_lib, bazel_exe, autotools_lib, autotools_exe. E.g. 'conan new cmake_lib -d name=hello -d version=0.1'. You can define
                        your own templates too by inputting an absolute path as your template, or a path relative to your conan home folder.
```

Reply:
It's because it's not generated from this list https://github.com/conan-io/conan/blob/f7f99b1fb964620864f2b2e598049ef6a6488c26/conan/cli/commands/new.py#L19 :)